crates/iota/src/client_ptb/ptb.rs

```diff
@@ -37,6 +38,12 @@ use crate::{
 pub struct PTB {
     #[arg(trailing_var_arg = true, allow_hyphen_values = true)]
     pub args: Vec<String>,
+    /// Select which fields of the response to display.
+    /// If not provided, all fields are displayed.
+    /// The fields are: effects, input, events, object_changes,
```

Comment:
What is the `input` field?

Reply:
Input for the transaction

Reply:
singular?

Reply:
Reverted as it's also called like this in the IotaTransactionBlockResponseOptions